x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_creation/constants.ts

```diff
@@ -9,6 +9,34 @@ import { i18n } from '@kbn/i18n';
 
 export const DEFAULT_LANGUAGE = 'Universal';
 
+export const ENGINE_CREATION_SELECT_APP_SEARCH_TITLE = i18n.translate(
```

Comment:
Not a big one but for the sake of consistency we use inline translations instead of constant files.